### DISCUSSION TITLE
[博客封面] Next主题读取的博客图片模糊有点模糊

### DISCUSSION THREAD

**Discussion by 7sasuke7l** [tangly1024/NotionNext#1983] [Q&A]
**描述bug**
Next主题下，Notion的封面图模糊

**复现步骤**

**期望的正常结果**
希望能将封面图显示清晰

**截图**
原Notion图片
![image](https://github.com/tangly1024/NotionNext/assets/50908165/a40a3acf-59f3-4424-bb24-810ffe92ef1c)
博客图片
![image](https://github.com/tangly1024/NotionNext/assets/50908165/5ef1ad98-ae18-4175-9f14-8c9092581348)

**环境**

- 操作系统: [windows]
- 浏览器 [Chrome]
- NotionNext版本 [4.05]
- 主题 [Next]

**补充说明**
与问题相关的其它说明

**Comment by tangly1024** [comment]
为了提升网页加载性能，封面图片做了压缩处理，从而减小图片大小。这是一个不得已的牺牲，同时保证画质清晰，且网站不卡顿，也许有更好的方案？

**Comment by Miing3016** [comment]
> 为了提升网页加载性能，封面图片做了压缩处理，从而减小图片大小。这是一个不得已的牺牲，同时保证画质清晰，且网站不卡顿，也许有更好的方案？

是否可以加入选项，选择是否压缩图片

**Reply by luras-oil** [reply]
感觉封面图不清晰会导致网站的浏览体验下降很多。支持能增加一个选项让用户自己选择。

**Reply by Ultronxr** [reply]
> 感觉封面图不清晰会导致网站的浏览体验下降很多。支持能增加一个选项让用户自己选择。

目前的解决办法，把图片上传到 图床 或 OSS 或 CDN，然后在 Notion 配置里面使用图片链接即可绕过压缩。
![Snipaste_2024-07-24_00-28-15](https://github.com/user-attachments/assets/70b55f7e-bbce-4d82-9f09-7fe66e90367d)
